packages/cf.js/src/legacy/utils/index.ts

```diff
@@ -2,7 +2,7 @@ import ethers from "ethers";
 import * as _ from "lodash";
 
 import * as abi from "./abi";
-import { FreeBalance } from "./free-balance";
+import { EMPTY_FREE_BALANCE, FreeBalance } from "./free-balance";
```

Comment:
Seems like a weird choice to export this like this. Shouldn't it be in a `constants` subdirectory like `ethers.constants`?

Reply:
Yeah that's better.